samples/grids/data-grid/cell-editing/App.razor

```diff
@@ -77,14 +77,14 @@
 
     private bool ButtonsDisabled { get; set; }
 
-    private List<Employee> Data;
+    public static System.Collections.ObjectModel.ObservableCollection<Employee2> Data { get; set; }
```

Comment:
add "@using System.Collections.ObjectModel" at start of this file
remove "System.Collections.ObjectModel." from the Data property

rename "Employee2" to "ObsEmployee"